modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/site_settings/site_url.jsp

```diff
@@ -152,7 +153,7 @@ if (privateVirtualHostnames.isEmpty()) {
 				cssClass="lfr-form-row"
 			>
 				<clay:row>
-					<aui:input inlineField="<%= true %>" label="public-pages" maxlength="200" name="publicVirtualHostname[]" placeholder="virtual-host" type="text" value="<%= virtualHostname %>" wrapperCssClass="col-sm-6" />
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: